src/html/macros/helpers.html

```diff
@@ -0,0 +1,6 @@
+{% macro sprite(id, viewBox, classes) %}
+  {% set viewBox = viewBox or "0 0 1 1"%}
+  <span class='sprite -{{id}} {{classes}}'>
+    <svg viewBox="{{viewBox}}"><use xlink:href='images/icons.svg#{{id}}'></use></svg>
+  </span>
+{% endmacro %}
```

Comment:
Just a heads up, if you decide not to use svg sprites, then you could remove this macro, and the corresponding imports in your templates. This is just part of the base blendid setup. You could also disable the gulp task.